src/functions.js

```diff
@@ -626,19 +626,117 @@ defineFunction([
 defineFunction([
     "\\acute", "\\grave", "\\ddot", "\\tilde", "\\bar", "\\breve",
     "\\check", "\\hat", "\\vec", "\\dot",
-    // We don't support expanding accents yet
-    // "\\widetilde", "\\widehat"
+    "\\widehat", "\\widetilde", "\\overrightarrow", "\\overleftarrow",
+    "\\Overrightarrow", "\\overleftrightarrow", "\\overgroup",
+    "\\overlinesegment", "\\overleftharpoon", "\\overrightharpoon",
```

Comment:
Well, they both exist: ![](http://quicklatex.com/cache3/c5/ql_04c0508169fc7e47a49d9144de8d97c5_l3.svg)

The `\Overrightarrow` accent is pretty obscure. It comes from a package called `overrightarrow`. But the SVG file is already necessary for an extensible arrow. So this accent only costs a few lines of CSS.